libpod/common/common.go

```diff
@@ -61,22 +57,38 @@ func IsValidBool(str string) bool {
 	return IsTrue(str) || IsFalse(str)
 }
 
+// Helper function to determine the username/password passed
+// in the creds string.  It could be either or both.
+func parseCreds(creds string) (string, string) {
```

Comment:
I'm semi-actively trying to deprecate libpod/common as a package; can you throw this in cmd/podman/util.go instead?